src/sitemap/sitemaps.py

```diff
@@ -214,3 +220,48 @@ def __init__(self, region, language):
         self.queryset = self.queryset.filter(
             poi__in=self.region.pois.all(), language=self.language
         )
+
+
+class OfferSitemap(WebappSitemap):
+    """
+    This sitemap contains all urls to offers for a specific region.
+
+    :param priority: The priority of this sitemap's urls (``1.0``)
+    :type priority: float
+
+    :param queryset: The queryset of this sitemap
+    :type queryset: :class:`~django.db.models.query.QuerySet` [ :class:`~cms.models.offers.offer.Offer` ]
+
+    Parameters inherited from :class:`~sitemap.sitemaps.WebappSitemap`:
+
+    :param changefreq: The usual change frequency of this sitemap's urls (``monthly``)
+    :type changefreq: str
+    """
+
+    priority = 1.0
+    queryset = Offer.objects.all()
+
+    def __init__(self, region, language):
+        """
+        This init function filters the queryset of offers objects based on the given region.
+
+        :param region: The region of this sitemap's urls
+        :type region: ~cms.models.regions.region.Region
+
+        :param language: The language of this sitemap's urls
+        :type language: ~cms.models.languages.language.Language
+        """
+        # Instantiate WebappSitemap
+        super().__init__(region, language)
+        # Filter queryset based on region
+        self.queryset = self.queryset.filter(region=self.region)
+        self.language = language
+
+    def location(self, obj):
+        """
+        This location function returns the absolute path for a given object returned by items().
+
+        :param obj: Objects passed from items() method
+        :type obj: ~cms.models.offers.offer.Offer
+        """
+        return "/" + "/".join([obj.region.slug, self.language.code, "offers", obj.slug])
```

Comment:
```suggestion
        return "/" + "/".join([obj.region.slug, self.language.code, "offers", obj.slug])

    def sitemap_alternates(self, obj):
        return [
            {
                "location": f"{WEBAPP_URL}/{self.region.slug}/{language_tree_node.code}/offers/{obj.slug}",
                "lang_code": language_tree_node.code,
            }
            for language_tree_node in self.region.language_tree_nodes.filter(
                active=True
            ).exclude(language=self.language)
        ]
```
Just an idea how to add sitemap alternates for offers.

A bit of context to understand the `language_tree_nodes`: The language tree is a concept we introduced to manage translations in a pre-defined way (to make sure the initial content is added in the default language and the other translations follow the desired directions).